pkg/rancher-desktop/typings/electron-ipc.d.ts

```diff
@@ -217,6 +217,7 @@ export interface IpcRendererEvents {
   'dialog/populate': (...args: any) => void;
   'dialog/size': (size: { width: number; height: number }) => void;
   'dialog/options': (...args: any) => void;
+  'dialog/close': (...args: any) => void;
```

Comment:
The typing here is kinda bad. We could at least go for:
```suggestion
  'dialog/close': (args: { dialog: 'SnapshotsDialog', snapshotEventType: import('@pkg/main/snapshots/types').SnapshotEvent['type']} ) => void;
```
But yeah, that isn't quite correct because you wanted this to be generic (kind of)…